PS8lite.Rmd

```diff
@@ -186,11 +191,50 @@ MiscVal + MoSold + YrSold" %>%
 
 Fit one Random Forest model `model_rf` using any approach you want.
 
+## Fitting a `randomForest` via `caret`
+
+### Setup cross-validation
+
+The "tuning" parameter we're going to "optimize" over is the number of randomly chosen variables we use at each split in our CART trees: `mtry`. We're going to search over values 33 through 36 (the maximum number of predictor variables we have access to). 
+
 ```{r}
+# Define cross-validation settings: 10-fold CV
+fit_control <- caret::trainControl(method = "cv", number = 10)
 
+# Setup search grid of "tuning" parameters
+mtry <- 33:36
+tunegrid <- expand.grid(.mtry = mtry)
 ```
 
+### Perform cross-validation
+
+Note that this code chunk takes a few minutes to run. So we set this code chucks `cache = TRUE` to save the result when knitting the .Rmd file, so that future knits don't re-run this code block to save time. 
```

Comment:
Indeed! This is a great example of a problem where computational constraints become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!